css-basics_pokemon-page/css/styles.css

```diff
@@ -0,0 +1,116 @@
+/* The exercise will start here! Sometimes you may need to add a class attribute to some elements in your HTML structure, but otherwise it won't be necessary to adjust the HTML to solve the tasks. */
+
+/* ------- General Styles ------- */
+
+/* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
+
+/* add selector */
+* {
+  font-family: monospace;
+  font-size: 1rem;
+  line-height: 1.5;
+  background-color: LemonChiffon;
+}
+
+/* ------- Headings Part 1 ------- */
+
+/* Task 3: To set the line height to 1.5 is good for texts, because of accessibility concerns, but for the headings it seems too much. Let's change this by giving our headings a line height of 1.2: */
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```

Comment:
If I'm not mistaken there's only the h1 and h2 selectors. :)

Reply:
done